plugins/kubeval

```diff
@@ -0,0 +1 @@
+repository = https://github.com/stefansedich/asdf-kubeval
```

Comment:
This URL should end with `.git` as shown in the Github clone dialog.

Reply:
@Stratus3D that is fixed!